docs/semgrep-cloud-platform/getting-started.md

```diff
@@ -94,6 +76,95 @@ Using Semgrep Supply Chain with Semgrep Cloud Platform provides users with the V
+![DESCRIPTION](/img/source-code-manager.png#md-width)
+8. Follow the steps to connect your source code manager. You can review permissions needed by Semgrep in [Requested permissions for GitHub and GitLab](#requested-permissions-for-github-and-gitlab).
+9. Optional: Refer to the instructions in [SSO configuration](/semgrep-cloud-platform/sso/) to set up SSO for your organization.
+
+## Joining an organization
+
+:::info Prerequisites
+- Your admin or person inviting you to Semgrep Cloud Platform should have an existing organization (org) in SCP that is connected to an org in either GitHub or GitLab.
+- You must be a member of the GitHub or GitLab org that is connected to Semgrep.
+:::
+
+For **members**, perform the following steps to join the org:
+
+1. Sign in to Semgrep Cloud Platform. If your administrator has configured SSO, click **Use SSO**. A **Welcome to Semgrep** dialog box appears.
+2. Check that the organization under the **Join an organization** button is the correct org to join. If there is more than 1 org, select the org to join.
+![DESCRIPTION](/img/join-an-org.png#md-width)
+3. Click **Join an organization**.
+
+## Running scans
+
+### Starting a SAST and SCA scan on a remote repository
+
+<PlatformAddRepo />
+
+#### Detecting GitHub repositories
+
+<PlatformDetectGhRepos />
+
+:::tip Product-specific information
+* To learn more about SAST scans on your codebase, see [Getting started with Semgrep Code](/semgrep-code/getting-started).
+* To learn more about SCA scans for your third-party dependencies, see [Getting started with Semgrep Supply Chain](/semgrep-supply-chain/getting-started).
+* Both products are **free for up to 10 contributors**. See [Usage limits](/usage-limits) to learn more about contributors and usage limits.
+:::
+
+### Starting a local repository scan and sending findings to SCP
+
+You can send scan results from a local repository to Semgrep Cloud Platform. The local repository is a separate **Project** from its remote counterpart.
+
+:::caution
+For team members of an organization, it is **not** recommended to send local repository scan results to the organization account. When logging in to Semgrep from the CLI through `semgrep login`, ensure that you are signed in SCP as your **personal account**. See [Projects separation between local and remote repositories](#projects-separation-between-local-and-remote-repositories).
+:::
+
+To send findings from a local repository, perform the following steps:
+
+1. Ensure that you are signed into Semgrep Cloud Platform in the account you want to send findings to. It is recommended to send local repository findings to your **personal** account.
+2. Log in to Semgrep:
+```
+semgrep login
+```
+2. Click the login URL provided. Your are taken to your web browser to complete the login process.
+3. Follow any additional steps.
+4. After logging in, start a scan in your CLI:
+```
+semgrep ci
+```
+
+#### Projects separation between local and remote repositories

```

Comment:
```suggestion
#### Project separation between local and remote repositories
```
Projects sounds a bit odd to me. If we do change this, the link above would need to be changed too.